docs/03-conventions.md

```diff
@@ -1,19 +1,33 @@
 # Conventions
 ## Naming
-For naming, we follow the airbnb style: https://github.com/airbnb/javascript/tree/master/react
+We follow the [airbnb style](https://github.com/airbnb/javascript/tree/master/react) for naming.
 
 ## Code style
-For the JavasScript code style we use the standard style: https://standardjs.com/
+We follow the [standard style](https://standardjs.com/) for JavaScript.
```

Comment:
This is pretty much outdated; afacs we don't follow these conventions.

Reply:
We actually mainly follow the rules as far as i could see. Of course there may be smaller deviations. Adjusted the docs.